src/test/java/cloud/fogbow/ras/core/plugins/interoperability/openstack/publicip/v2/OpenStackPublicIpPluginTest.java

```diff
@@ -1,576 +1,1231 @@
-		Mockito.doThrow(badRequestException).when(this.httpClient).doGetRequest(
-				Mockito.anyString(), Mockito.any(OpenStackV3User.class));
-
-		// exercise
-		try {
-			this.openStackPublicIpPlugin.getInstance(publicIpOrder, this.openStackV3Token);
-			Assert.fail();
-		} catch (InvalidParameterException e) {}
-
-		// verify
-		Mockito.verify(this.httpClient, Mockito.times(1)).doGetRequest(
-				Mockito.anyString(),
-				Mockito.eq(this.openStackV3Token));
-	}
-
-
-    private String getPortsResponseJson(String... portsId) {
-    	List<Map<String, Object>> idsJsonKey = new ArrayList<Map<String, Object>>();
-
-    	for (int i = 0; i < portsId.length; i++) {
-    		Map<String, Object> idJsonKey = new HashMap<String, Object>();
-    		idJsonKey.put(OpenStackConstants.PublicIp.ID_KEY_JSON, portsId[i]);
-    		idsJsonKey.add(idJsonKey);
-		}
-
-        Map<String, Object> portsJsonKey = new HashMap<String, Object>();
-        portsJsonKey.put(OpenStackConstants.PublicIp.PORTS_KEY_JSON, idsJsonKey);
-
-        Gson gson = new Gson();
-        return gson.toJson(portsJsonKey);
-	}
-
-	private String getCreateFloatingIpResponseJson(String floatingIpId) {
-    	Map<String, Object> idJsonKey = new HashMap<String, Object>();
-    	idJsonKey.put(OpenStackConstants.PublicIp.ID_KEY_JSON, floatingIpId);
-        Map<String, Object> floatingipJsonKey = new HashMap<String, Object>();
-        floatingipJsonKey.put(OpenStackConstants.PublicIp.FLOATING_IP_KEY_JSON, idJsonKey);
-
-        Gson gson = new Gson();
-        return gson.toJson(floatingipJsonKey);
-    }
-
-	private String getGetFloatingIpResponseJson(String floatingIpId,
-			String floatingIpAddress, String status) {
-    	Map<String, Object> idJsonKey = new HashMap<String, Object>();
-    	idJsonKey.put(OpenStackConstants.PublicIp.ID_KEY_JSON, floatingIpId);
-    	idJsonKey.put(OpenStackConstants.PublicIp.FLOATING_IP_ADDRESS_KEY_JSON, floatingIpAddress);
-    	idJsonKey.put(OpenStackConstants.PublicIp.STATUS_KEY_JSON, status);
-        Map<String, Object> floatingipJsonKey = new HashMap<String, Object>();
-        floatingipJsonKey.put(OpenStackConstants.PublicIp.FLOATING_IP_KEY_JSON, idJsonKey);
-
-        Gson gson = new Gson();
-        return gson.toJson(floatingipJsonKey);
-    }
-
-	private String createFloatingIpRequestBody(String floatingNetworkId, String projectId, String portId) {
-		CreateFloatingIpRequest createBody = new CreateFloatingIpRequest.Builder()
-				.floatingNetworkId(floatingNetworkId)
-				.projectId(projectId)
-				.portId(portId)
-				.build();
-
-		return createBody.toJson();
-	}
-
-	public String getCreateSecurityGroupResponseJson(String id) {
-		CreateSecurityGroupResponse.SecurityGroup securityGroup = new CreateSecurityGroupResponse.SecurityGroup(id);
-		return new CreateSecurityGroupResponse(securityGroup).toJson();
-	}
-
-	private ListSecurityGroups createListSecurityGroupsResponse() {
-		ListSecurityGroups.SecurityGroup securityGrou
```

Comment:
There's no need for this mock since the test does not depend on the json being returned.

Reply:
Okay, I'll remove it.